pkg/antctl/raw/check/installation/command.go

```diff
@@ -305,6 +325,39 @@ func (t *testContext) setup(ctx context.Context) error {
 	return nil
 }
 
+func (t *testContext) runTests(ctx context.Context) testStats {
+	var stats testStats
+	for name, test := range testsRegistry {
+		if t.runFilterRegex != nil && !t.runFilterRegex.MatchString(name) {
+			continue
+		}
+		t.Header("Running test: %s", name)
+		if err := test.Run(ctx, t); err != nil {
+			if errors.As(err, new(notRunnableError)) {
+				t.Warning("Test %s was skipped: %v", name, err)
+				stats.numSkipped++
+			} else {
+				t.Fail("Test %s failed: %v", name, err)
+				stats.numFailure++
+			}
+		} else {
+			t.Success("Test %s passed", name)
+			stats.numSuccess++
+		}
+	}
+	return stats
+}
+
+func (t *testContext) runAgnhostConnect(ctx context.Context, clientPodName string, container string, target string, targetPort int) error {
+	cmd := agnhostConnectCommand(target, fmt.Sprint(targetPort))
+	_, stderr, err := check.ExecInPod(ctx, t.client, t.config, t.namespace, clientPodName, container, cmd)
+	if err != nil {
+		// We log the contents of stderr here for troubleshooting purposes.
+		t.Log("/agnhost command failed - stderr: %s", stderr)
```

Comment:
Maybe including the full cmd can also help here?

Reply:
I think it's ok right now because the command is always the same, and we have enough context from the surrounding log lines. If we add more complex commands, we can start logging them as well.